src/components/SidePanel/index.tsx

```diff
@@ -1,10 +1,13 @@
 import React from 'react';
 import useSidePanel from '@hooks/useSidePanel';
 import Help from './HelpModal';
+import useSyncSidePanelWithHistory from './useSyncSidePanelWithHistory';
 
 function SidePanel() {
     const {isSidePanelTransitionEnded, shouldHideSidePanel, sidePanelTranslateX, shouldHideSidePanelBackdrop, closeSidePanel} = useSidePanel();
 
+    useSyncSidePanelWithHistory();
```

Comment:
Can you please add a comment about this? why it is added here specifically and what it does?